content/en/docs/community/release_checklist.md

```diff
@@ -389,7 +391,9 @@ created by running the following command:
 
 ```shell
 export PREVIOUS_RELEASE=vX.Y.Z
+make fetch-dist
 make release-notes
+make sign
```

Comment:
This is covered in the signing section. You should run `make clean` first. The fetch the dist. Then go to the signing section. clean and fetch-dist only need to be run in the signing section if not already run.

Reply:
Thanks. I've attempted to clarify.